.github/workflows/pull_request_review.yml

```diff
@@ -16,12 +17,20 @@ jobs:
 
     strategy:
       matrix:
-        node: [10.x]
+        node: [12.x]
```

Comment:
The only reason why I had kept it at 10 was due to cli

Reply:
Seems like it works though, even on the latest release... and this is ready for 3.0 as well.